docs/creating-a-kubernetes-cluster.md

```diff
@@ -90,7 +90,7 @@ For example:
 ```shell
 eval $(minikube docker-env)
 docker pull gcr.io/knative-samples/primer:latest
-docker tag gcr.io/knative-samples/primer:latest dev.local/knative-samples/primer:v1
+docker tag gcr.io/knative-samples/primer:latest ko.local/knative-samples/primer:v1
```

Comment:
🤔 but the image wasn't built with `ko`...

Reply:
I'm a bit confused.   In order to instruct ko to sideload images into Docker daemon, we set `KO_DOCKER_REPO=ko.local`.   But in the next paragraph, to have Knative access an image in Minikube's Docker daemon, we would reference image with `dov.local` and not `ko.local`?

Reply:
`ko.local` has special significance to `ko`, and it just seems weird to me to use the `ko` sentinel in place of another equally valid non-`ko` sentinel to keep our controllers from resolving non-`ko`-built image tags to digests.

Reply:
It sounds like the doc is good as is.  I 'm closing this PR.